handbook/engineering/core-application/areas-of-ownership.md

```diff
@@ -0,0 +1,35 @@
+# Core Application areas of ownership
+The areas of ownership are organized base on different work-streams that the Core Application team is currently focusing on.
+
+## Code host connections and repositories management
+End-to-end experience managing code host connections and repositories for all supported version control systems, including database storage, back-end service, and UI layer.
+
+## Licensing and subscription management
+- License generation and enforcement
+- Pricing and packaging
+- Billing and payments
+- Subscription management
+- Usage reporting
+
+## Administration experience
+Provide transparency and control to how Sourcegraph is being used with our customers' organizations, teams, and individual setups.
+These includes:
+
+- Site administrator experience
+- Instance and user configuration
+- User management
+- Organisations
+- Role-based access control
+- Audit logs 
+
+## Identity and access management
```

Comment:
The bullets below here should be captitalised

Reply:
fixed